src/docs/src/ddocs/mango.rst

```diff
@@ -0,0 +1,794 @@
+.. Licensed under the Apache License, Version 2.0 (the "License"); you may not
+.. use this file except in compliance with the License. You may obtain a copy of
+.. the License at
+..
+..   http://www.apache.org/licenses/LICENSE-2.0
+..
+.. Unless required by applicable law or agreed to in writing, software
+.. distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+.. WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+.. License for the specific language governing permissions and limitations under
+.. the License.
+
+=============
+Mango Queries
+=============
+
+In addition to :ref:`map/reduce views <viewfun>`, CouchDB supports a expressive
+query syntax called Mango.
+
+The mango system consists of two major concepts:
+
+* Selectors, which are the queries, and are passed to the mango endpoints
+* Indexes, which are a specialization of design docs used in mango queries
```

Comment:
```suggestion
* Selectors, which are the queries, are passed to the mango endpoints and
* Indexes, which are a specialization of design docs used in mango queries
```

Reply:
I personally don't like this as a writing style. Certainly not without punctuation.